.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,7 @@
+contact_links:
+  - name: Adafruit Forum
+    url: https://forums.adafruit.com/
+    about: Official Adafruit technical support forum. Good for getting help on getting a project working.
+  - name: Adafruit Discord
+    url: https://adafru.it/discord
+    about: Unofficial chat with many helpful folks and normally prompt replies.
```

```suggestion
    about: Official chat group, with many helpful folks and normally prompt replies.
```
I believe the Discord is official.

---

This was suggested by @tannewt [above](https://github.com/adafruit/circuitpython/pull/4316#discussion_r586593147).

---

Ooh, didn't see that.